File: C_PROGRAMMING/C_PROGRAMS/Statements/2_practical/program5.c

// WAP to print the spelling of given number...
//
#include<stdio.h>
void main()
{
  int num;
  printf("\n Enter the number :\n");
  scanf("%d",&num);
  
  switch(num)
  {
    case 0:
       printf("\n zero\n");
       break;

    case 1:
       printf("\n One\n");
       break;

   case 2:
       printf("\n Two\n");
       break;

   case 3:
       printf("\n Three\n");
       break;

   case 4:
       printf("\n Four\n");
       break;
  
   case 5:
       printf("\n Five\n");
       break;

  deafult:
       printf("\n You entered greater than 5\n\n");
  }

}

File: JAVA_PROGRAMMING/CLASS_CODES/Control_Statements/For_loop/program5.java

// WAP print factors of a given number and count
class Demo{

	public static void main(String[] args){
	
		int num = 20;
		int count =0;
		
		for(int i =1; i <= num ; i++){

			if(num % i == 0){
				count++;
				System.out.println(i);
			}

		}
		System.out.println("Count of factors is  : "+count);

	}
		
}
